ansible/roles/debops.dovecot/defaults/main.yml

```diff
@@ -718,6 +760,10 @@ dovecot__ldap_start_tls: '{{ ansible_local.ldap.start_tls
 # .. envvar:: dovecot__ldap_user_filter [[[
 #
 # The LDAP filter used to look up user accounts in the directory.
+#
+# See `LDAP ACLs - Known access controls
+# <https://docs.debops.org/en/master/ansible/roles/debops.ldap/ldap-access.html#known-access-controls>`_
```

Comment:
This should be changed to an internal reference in the documentation.

Reply:
Done

Reply:
Done